test/gst-vaapi/vpp/brightness.py

```diff
@@ -24,9 +24,15 @@ def test_default(case, level):
     "{}_brightness_{level}_{format}_{width}x{height}"
     ".yuv".format(case, **params))
 
+  if os.environ.get("LIBVA_DRIVER_NAME", None):
+    params.update(vpp_format = iHD_input_format(params["format"]))
+  else:
+    params.update(vpp_format = params["format"])
+
```

Comment:
With aforementioned recommendation in util.py, this can be reduced to:
```
hwup_format = mapformat_hwup(params["format"])
```
... and just add it to the first `params.update(...)` call.